pa1/PPTokenizerDFA.cpp

```diff
@@ -0,0 +1,1386 @@
+#include "PPCodePointCheck.h"
+#include "PPCodeUnitCheck.h"
+#include "PPTokenizerDFA.h"
+#include <assert.h>
+
+// TODO: Create a common utility debug logging tool instead of this macro
```

Comment:
Yes, I want to do this. But that should be a separate pull request. This pr focuses on preprocessing tokenizer, not utilities as much.